blog-service/2024-01-11-search.md

```diff
@@ -0,0 +1,20 @@
+---
+title: New Visualizations (Search) 
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - log_level
```

Comment:
Should we have keywords related to Aggregate charts here ??

Reply:
@ssharma-sumo @JV0812 should aggregate charts also be mentioned in the body? It looks like in the screenshot, the "Aggregates" tab is selected.

Reply:
@JV0812 bump ☝️

Reply:
added Aggregates as a new keyword.